static/css/components/footer.less

```diff
@@ -123,3 +123,45 @@ div#version-details {
   border-radius: 3px;
   padding: 8px 10px;
 }
+
+.dropbtn {
```

Comment:
This feature doesn't work without javascript from what I can see and footer.less is render  blocking. Styles should be loaded inside all.js.

Reply:
Okay, but from what I have noticed that Header (Browse, More, and Account component) does not support keyboard navigation so is there a way to implement it in the current structure?

Reply:
If not I guess we should make this dropdown keeping keyboard navigation in mind.
Later we can implement the same for other dropdown menus in the header.